### PR TITLE
Fix image widget sizing issues

### LINKED ISSUE #2066: Image items are not refreshing
### Actual behaviour

In 2.13.5-beta `Image` items with refresh interval set are not refreshing. The issue is not present in 2.13.0

### Expected behaviour
Image items should refresh properly.

 
### Steps to reproduce
1. Add an Image item with refresh interval (eg camera feed) to the sitemap
2. Open the sitemap in the app

The issue has probably originated in #2039 but I'm not sure.

### LINKED ISSUE #2118: Image widget is higher than device height
### Actual behaviour
Several SVG icons without explicit size are displayed with a higher height than the screen height, when used in image widget.

This diff fixes the issue, but maybe it should be investigated why this happens.

````diff
diff --git a/mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt b/mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt
index cd3c1dca..90e7edfd 100644
--- a/mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt
+++ b/mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt
@@ -287,9 +287,9 @@ class WidgetImageView constructor(context: Context, attrs: AttributeSet?) : AppC
                         .asBitmap(size, conversionPolicy)
                         .response
                     if (adjustViewBoundsForDownscalingOnly) {
-                        // make sure that view only shrinks to accomodate bitmap size, but doesn't enlarge ... that is,
+                        // make sure that view only shrinks to accommodate bitmap size, but doesn't enlarge ... that is,
                         // adjust view bounds only if width is larger than target size
-                        adjustViewBounds = bitmap.width > size
+                        adjustViewBounds = bitmap.width >= size
                     }
                     setBitmapInternal(bitmap)
                     CacheManager.getInstance(context).cacheBitmap(url, bitmap)

````

### Expected behaviour
Image widgets should honor https://github.com/openhab/openhab-android/blob/master/mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt#L620
 
### Steps to reproduce
1. Add this to your sitemap: `Image url="http://openhab/icon/switch?state=UNDEF&format=svg" label="foo"`
2. Rotate device into landscape
3. Open the subpage with this image widget

#### Client
* App version : 2.13.18-beta

### PR DESCRIPTION
**Pull request by maniac103**
Closes #2066 
Closes #2118

### CODE REVIEW

File: mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt [hunk 1]
```diff
@@ -231,6 +222,20 @@ class WidgetImageView constructor(context: Context, attrs: AttributeSet?) : AppC
         pendingLoadJob = null
     }
 
+    private fun applyLoadedBitmap(bitmap: Bitmap) {
+        removeProgressDrawable()
+        if (adjustViewBoundsForDownscalingOnly) {
+            // Make sure that view only shrinks to accomodate bitmap size, but doesn't enlarge ... that is,
```
Comment (mueller-ma) [inline mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt:228]:
```suggestion
            // Make sure that view only shrinks to accommodate bitmap size, but doesn't enlarge ... that is,
```

Comment (maniac103) [reply mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.kt]: Fixed, thanks.

**Comment by mueller-ma** [comment]
#2118 is fixed for me with this PR, but images still stop auto refreshing after rotation.

**Comment by maniac103** [comment]
> but images still stop auto refreshing after rotation.

Right, not sure how I missed that. Should be fixed now.

**Pull request merged**